sopel/config/core_section.py

```diff
@@ -777,9 +777,10 @@ def homedir(self):
     host_blocks = ListAttribute('host_blocks')
     """A list of hostnames which Sopel should ignore.
 
-    Messages from any user whose connection hostname matches one of these
-    values will be ignored. :ref:`Regular expression syntax <re-syntax>`
-    is supported, so remember to escape special characters:
+    Sopel will :ref:`ignore <Ignoring Users>` messages from any user whose
```

Comment:
I would say not; "ignore" is a pretty generic term that's used in several different contexts. Tying it to this particular context doesn't make sense to me. (Different opinion entirely if you were proposing an alias to avoid breaking links should we choose to retitle the "Ignoring Users" section. But a reasonable alias would still require the `link text <actual-target>` syntax, so that too probably isn't worth amending the PR.)